src/types.js

```diff
@@ -62,9 +62,21 @@ export type DimensionFragment = {|
   center: Position,
 |}
 
+export type Placeholder = {|
```

Comment:
i am now adding this to every draggable. A bit redundant, but super useful and reducing the noise around memoization